snap/validate.go

```diff
@@ -310,13 +311,13 @@ func validateSocketAddrNetHost(socket *SocketInfo, fieldName string, address str
 		}
 	}
 
-	return fmt.Errorf("socket %q has invalid %q address %q", socket.Name, fieldName, address)
+	return fmt.Errorf("invalid %q address %q", fieldName, address)
```

Comment:
maybe add `(only addresses referring to localhost are valid)`? or is that explained elsewhere?